src/test/java/com/exasol/releasedroid/repository/MavenPomParserTest.java

```diff
@@ -47,38 +44,41 @@ void testParseMavenPomWithPlugins() throws IOException {
         final String pom = "<project>" //
```

Comment:
I don't like the manual indentation here.
But I also understand that you want to have it here and not as a resource.

In PK I created a Maven `Model` using its java API and write it to a pom file using the `MavenXpp3Writer`.
That allows also to build different test setups with just a few lines of code.